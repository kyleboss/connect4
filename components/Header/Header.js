import styled from "styled-components";
import { TABLET_AND_LARGER_SIZE } from "@/utils/constants";

const Header = styled.h1`
  font-size: 3rem;
  margin-bottom: 10px;
  font-weight: 800;
  text-align: center;

  @media (min-width: ${TABLET_AND_LARGER_SIZE}) {
    font-size: 4rem;
    margin-bottom: 20px;
  }
`;

export default Header;
