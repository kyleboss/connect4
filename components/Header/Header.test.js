import React from "react";
import { render } from "@testing-library/react";

import Header from "./Header";

describe("Header", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Header>Hello World</Header>);
    expect(asFragment()).toMatchSnapshot();
  });
});
