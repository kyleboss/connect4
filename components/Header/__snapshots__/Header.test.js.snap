// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<DocumentFragment>
  <h1
    class="sc-f9cd8f8e-0 dcLxvF"
  >
    Hello World
  </h1>
</DocumentFragment>
`;
